src/pages/home-page/HomePage.js

```diff
@@ -1,26 +1,43 @@
+import { useSelector } from "react-redux";
+
 import logo from "assets/logo.png";
 import LoginForm from "components/users/LoginForm/LoginForm";
 
 import "./home-page.scss";
 
-const HomePage = () => (
-  <main className="home-page">
-    <section className="login">
-      <figure>
-        <img src={logo} alt="Logo" className="logo" />
-      </figure>
-      <LoginForm />
-      <div className="forgot-password-link regular-text">
-        <a href="/">I forgot my password.</a>
-      </div>
-    </section>
-    <section className="signup">
-      <p className="regular-text">Don't have an account?</p>
-      <button type="button" className="common-white-button regular-text">
-        Sign up
-      </button>
-    </section>
-  </main>
-);
+const HomePage = () => {
+  const loading = useSelector((state) => state.user.loading);
+
+  const loggedInUser = sessionStorage.getItem("user");
+
+  return (
+    <main className="home-page">
+      {loggedInUser === null ? (
```

Comment:
Instead of comparing this variable with null you can use [Truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) and [Falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) 

```suggestion
      {!!loggedInUser ? (
```